homeassistant/components/weather/yandex.py

```diff
@@ -0,0 +1,218 @@
+"""
+Support for Yandex Weather.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/weather.yandex/
+"""
+import logging
+from datetime import datetime, timedelta
+from contextlib import wraps
+
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.components.weather import (
+    WeatherEntity, PLATFORM_SCHEMA, ATTR_FORECAST_CONDITION,
+    ATTR_FORECAST_PRECIPITATION, ATTR_FORECAST_TEMP,
+    ATTR_FORECAST_TEMP_LOW, ATTR_FORECAST_TIME)
+from homeassistant.const import \
+    CONF_NAME, TEMP_CELSIUS, CONF_LATITUDE, CONF_LONGITUDE, CONF_API_KEY
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers import config_validation as cv
+
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['yandex-weather-api==0.2.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = 'Яндекс.Погода'
```

Comment:
Should be in English.

Reply:
Ok. [English API docs](https://tech.yandex.com/weather/doc/dg/concepts/pricing-docpage/#about) even state that 'Yandex.Weather' is the recommended way to that. Is there any way to translate it in the UI/?

Reply:
The source language should be English. Thus we can use "According to Yandex.Weather".